src/Commands/Configure.ts

```diff
@@ -45,11 +35,25 @@ export class Configure implements CommandModule {
         describe: `Enables network tests only`
       })
       .middleware((args: Arguments) => {
+        const { api, bus } = Helpers.getClusterUrls(args);
+
         container.register(ConnectivityUrls, {
           useValue: new Map(
             Object.values(TestType).map((type: TestType) => {
               try {
-                return [type, new URL(args[type] as string)];
+                switch (type) {
+                  case TestType.TCP:
+                    return [type, new URL((args[type] as string) ?? bus)];
+                  case TestType.AUTH:
+                    return [
+                      type,
+                      new URL(
+                        (args[type] as string) ?? `${api}/v1/repeaters/user`
```

Comment:
Wrong URL, `auth` must be `${cluster}/v1/repeaters/user`

Reply:
its same thing, cluster overrides api or getting default value 'https://nexploit.app'

Reply:
not relevant